src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -464,8 +465,12 @@ Statement SingleStatement() :
         LOOKAHEAD(CreateView())
         stm = CreateView()
         |
+        LOOKAHEAD(AlterView())
```

Comment:
I believe this is needed since we now have 2 productions that match on `K_ALTER`, so we need to peek and check if it is one or the other